src/surveys/components/DateResponse.tsx

```diff
@@ -0,0 +1,64 @@
+import { h } from 'preact';
+import { Button, Header } from '../../components';
+import { AnswerType, ID, Question, SurveyAnswer, Theme } from '../../types';
+
+import { useState } from 'preact/hooks';
+import { Value } from 'react-date-picker/dist/cjs/shared/types';
+import { DatePicker } from '../../components';
+
+interface DateResponseProps {
+  question: Question;
+  onAnswered: (questionId: ID, answer: SurveyAnswer) => void;
+  submitText?: string;
+  theme?: Theme;
+}
+
+export default function DateResponse({
+  question,
+  onAnswered,
+  submitText,
+  theme,
+}: DateResponseProps) {
+  const [selectedDate, setSelectedDate] = useState<Value>();
+
+  const handleSubmit = (e: h.JSX.TargetedEvent<HTMLFormElement, Event>) => {
+    e.preventDefault();
+    if (!selectedDate) return;
+    onAnswered(question.id, {
+      type: AnswerType.DATE,
+      value: selectedDate.toLocaleString(),
+    });
+  };
+
+  return (
+    <form className='flex flex-col gap-4' onSubmit={handleSubmit}>
+      <div className='mr-6'>
+        <Header title={question.label} color={theme?.question} />
+      </div>
+
+      <label htmlFor={`date-${question.id}`} className={'flex flex-col gap-2'}>
+        <span style={{ color: theme?.answer ? theme.answer : '#050505' }}>
+          {question.description ?? 'Date'}
```

Comment:
The description here should be a `Header` like other response types. It's not a label